### PR TITLE
Update Readme, Changelog, SPM docs

### PR DESCRIPTION
**Pull request by aashishpatil-g**
Update Readme, Changelog, SPM docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,161 +16,8 @@ Instructions for [Swift Package Manager](https://swift.org/package-manager/) sup
 found in the [SwiftPackageManager.md](SwiftPackageManager.md) Markdown file.
 
 ## Getting Started
-Firebase Data Connect is in Private Preview at no cost for a limited time. Sign up the program at https://firebase.google.com/products/data-connect.
-
-Once you are selected as an allowlist member, you should be able to create a Cloud SQL instance through Firebase Data Connect console.
-
 Here's a quick rundown of steps to get you started. Learn more about details at the official [Getting Started documentation](https://firebase.google.com/docs/data-connect/quickstart).
```
Comment (paulb777) [inline README.md:19]: Delete "Here's a quick rundown of steps to get you started. "

**Review by dconeybe — APPROVED** [review]

File: Sources/QueryRef.swift [hunk 2]
```diff
@@ -122,10 +122,10 @@ public protocol ObservableQueryRef: QueryRef {
   var lastError: DataConnectError? { get }
 }
 
-// QueryRef class used with ObservableObject protocol
-// data: Published variable that contains bindable results of the query.
-// lastError: Published variable that contains DataConnectError if last fetch had error.
-//            If last fetch was successful, this variable is cleared
+/// QueryRef class used with ObservableObject protocol
+/// data: Published variable that contains bindable results of the query.
+/// lastError: Published variable that contains DataConnectError if last fetch had error.
```
Comment (morganchen12) [inline Sources/QueryRef.swift:127]: The two parameters here don't match the type parameters. Also, symbols that should be links within the same module should be wrapped in triple backticks: 
```
```DataConnectError```
```

File: Sources/QueryRef.swift [hunk 3]
```diff
@@ -178,10 +178,10 @@ public class QueryRefObservableObject<
   }
 }
 
-// QueryRef class compatible with the Observation framework introduced in iOS 17
-// data: Published variable that contains bindable results of the query.
-// lastError: Published variable that contains DataConnectError if last fetch had error.
-//            If last fetch was successful, this variable is cleared
+/// QueryRef class compatible with the Observation framework introduced in iOS 17
+/// data: Published variable that contains bindable results of the query.
+/// lastError: Published variable that contains DataConnectError if last fetch had error.
```
Comment (morganchen12) [inline Sources/QueryRef.swift:183]: See above

File: Changelog.md [hunk 4]
```diff
@@ -1,2 +1,3 @@
-
+# 11.3.0-beta
+[Added] Initial public beta release of the SDK. For more information visit [Firebase Data Connect](https://firebase.google.com/products/data-connect)
```
[the remark this answers is not in the document]
Comment (aashishpatil-g) [reply Changelog.md]: Done.

File: Sources/QueryRef.swift (hunk 2 above)
Comment (aashishpatil-g) [reply Sources/QueryRef.swift]: They aren't type parameters. They are variables that this QueryRef publishes. 

I've updated docs for this and used the backticks to denote symbols. I've also documented the symbols within the class.

File: Sources/QueryRef.swift (hunk 3 above)
Comment (aashishpatil-g) [reply Sources/QueryRef.swift]: Same change as above

File: README.md (hunk 1 above)
Comment (aashishpatil-g) [reply README.md]: Fixed. thanks for the catch

File: Sources/QueryRef.swift [hunk 5]
```diff
@@ -17,11 +17,12 @@ import Foundation
 import Combine
 import Observation
 
+/// The type of publisher to use for the Query Ref
 @available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
 public enum ResultsPublisherType {
-  case auto // automatically determine ObservableQueryRef
-  case observableObject // pre-iOS 17 ObservableObject
-  case observableMacro // iOS 17+ Observation framework
+  case auto /// automatically determine ObservableQueryRef
```
Comment (paulb777) [inline Sources/QueryRef.swift:23]: Does this work when you build the docs in Xcode? I thought the comments needed to be on the line above?

File: SwiftPackageManager.md [hunk 6]
```diff
@@ -36,34 +28,8 @@ Then, select **Add Package**.
...
-eg. `scripts/upload-symbols`, and make sure that the file is executable:
-`chmod +x scripts/upload-symbols`.
-This script can be used to manually upload dSYM files (for usage notes and
-additional instructions, run with the `--help` parameter).
-
-If you're getting `error: Could not get GOOGLE_APP_ID in Google Services file from build environment` on the Crashlytics run script step and you're using Xcode 15 and specifically `User Script Sandboxing = YES`, make sure to include all input files referenced [here](https://github.com/firebase/firebase-ios-sdk/pull/11463) in the Crashlytics run script.
-
----
 
 ### Alternatively, add Firebase to a `Package.swift` manifest
```
Comment (paulb777) [inline SwiftPackageManager.md:34]: Firebase Data Connect

**Review by paulb777 — APPROVED** [review]

File: Sources/QueryRef.swift (hunk 5 above)
Comment (aashishpatil-g) [reply Sources/QueryRef.swift]: Thanks for helping out with the docs build. Yes, this seems to work except for the first enum case. I've changed it to where the comments are at the top.

File: SwiftPackageManager.md (hunk 6 above)
Comment (aashishpatil-g) [reply SwiftPackageManager.md]: Fixed

**Pull request merged**